README.md

```diff
@@ -1,39 +1,40 @@
+- The latest NVIDIA Deep Learning and LLM software libraries
+- Release notes for each of the NVIDIA optimized containers
+- Links to developer documentation
 
-## What are NVIDIA AI Workflows?
------------------------------
-NVIDIA AI Workflows are intended to provide reference solutions of how to leverage NVIDIA frameworks to build AI solutions for solving common use cases. These workflows provide guidance like fine tuning and AI model creation to build upon NVIDIA frameworks. The pipelines to create applications are highlighted, as well as opinions on how to deploy customized applications and integrate them with various components typically found in enterprise environments, such as components for orchestration and management, storage, security, networking, etc.
+## Retrieval Augmented Generation (RAG)
 
-By leveraging an AI workflow for your specific use case, you can streamline development of AI solutions following the example provided by the workflow to:
+A RAG pipeline embeds multimodal data --  such as documents, images, and video -- into a database connected to a Large Language Model.  RAG lets users to chat with their own data. 
 
-- Reduce development time, at lower cost
-- Improve accuracy and performance
-- Gain confidence in outcome, by leveraging NVIDIA AI expertise
+| Name                                                                                                                 | LLM   | Framework  | Multi-GPU | Multi-Node | Embedding | TRT-LLM | Triton                                                                                                    | VectorDB  | K8s                                                                                                                                          |
+|------------------------------------------------------------------------------------------------------------------------|-------------|------|-----------|------------|----------|------|-----------------------------------------------------------------------------------------------------------|------|---------------------------------------------------------------------------------------------------------------------------------------------|
+| [Developer RAG](https://github.com/NVIDIA/GenerativeAIExamples/tree/main/RetrievalAugmentedGeneration)                       | llama2-13b     | Langchain  | Yes       | No        | e5large-v2| Yes    | Yes    | Milvus  | No                                                                                                                                           |
+| [Developer RAG](https://github.com/NVIDIA/GenerativeAIExamples/tree/main/RetrievalAugmentedGeneration)                       | llama2-13b     | Llama Index  | Yes       | No        | e5large-v2| Yes    | Yes    | Milvus  | No                                                                                                                                           |
```

Comment:
We have a single RAG example. We should have single row in this table with framework set as Langchain+Llama Index.
Embedding should be e5-large-v2.

Reply:
okay updated